demo/components/tree-view-demo.st.css

```diff
@@ -22,15 +23,15 @@
 }
 
 .tree-node:hover {
-    border: 1px solid value(color_treeFocused);
+    border: 1px solid value(color_Keyboard_Focused);
```

Comment:
Since you are basically repeating the selectors in the default tree-node.st.css, why not extend it? Best thing to do is to extend and add a custom flare to it

Reply:
after discussion we agreed the demo classNames are unrelated to the default tree node classNames (and its only by chance that they are the same as they could be totally different because they are provided from the outside through `itemRenderer`)